plasma_framework/Makefile

```diff
@@ -1,6 +1,10 @@
 PYTHON_TESTS_DIR = python_tests/
 RUN_PYTHON_MAKEFILE_COMMAND = $(MAKE) -C $(PYTHON_TESTS_DIR)  
 
+.PHONY: list
```

Comment:
switch between projects I forget which receipts I have available so this little 
`make list` helps me so that I don't have to open Makefile... If you want it I can leave it otherwise I can delete it too

Reply:
Nice, leave it in 😄